content/view/view.html

```diff
@@ -11,8 +11,8 @@ <h4 id="publicProfileUsername">{{username}}</h4>
 		{{profile}}
 		<hr>
 	</div>
-	<div ng-show="!user && !error" class="do-not-animate">
-		<img id="spinner" src="image/spinner.gif" class="fadeFast">
+	<div ng-show="!user && !error" style="text-align: center" class="do-not-animate">
+		<img class="spinner" src="image/spinner.gif" class="fadeFast">
```

Comment:
I actually mean: http://stackoverflow.com/questions/9512330/multiple-class-attributes-in-html